README.md

```diff
@@ -25,7 +25,7 @@ Important: If you are using [auto-instrumentation](#auto-instrumentation), you s
 
 2. Build the OpenTelemetry `BatchSpansProcessor` with the `NewRelicSpanExporter` 
 ```java
-    BatchSpansProcessor spanProcessor = BatchSpansProcessor.create(exporter);
+    BatchSpanProcessor spanProcessor = BatchSpanProcessor.newBuilder(exporter).build();
```

Comment:
I think `builder()` is more idiomatic than `newBuilder()` but don't feel too strongly about it.

Reply:
Hah, commented this before realizing that this is from upstream.  Derp.  I should have made this comment over there weeks ago it seems.  Heh.

Reply:
months, actually. :)

Reply:
For reasons I don't know, OTel has standardized on "newBuilder"